### DISCUSSION TITLE
v0.1.0

### DISCUSSION THREAD

**Discussion by ericdrosas87** [lsst-epo/citizen-science-notebooks#73] [Announcements]
## What's Changed
* Epo 6783 by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/1
* Adding DP0.2 cutout option by @clareh in https://github.com/lsst-epo/citizen-science-notebooks/pull/2
* Updated notes and added debugging by @clareh in https://github.com/lsst-epo/citizen-science-notebooks/pull/3
* Removed duplicate info messages, now added steps by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/4
* Abstracted away citSci SDK to its own notebook by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/5
* Changed logging style of errors and warnings by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/6
* Fixed SDK notebook bug that caused a stack trace by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/7
* Added functionality for creating manifest and by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/8
* Added changelog and updated SDK notebook by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/11
* Utils by @beckynevin in https://github.com/lsst-epo/citizen-science-notebooks/pull/33
* Added flexible manifest creation/modification by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/36
* Fetch classifications by @beckynevin in https://github.com/lsst-epo/citizen-science-notebooks/pull/39
* Removed unnecessary data type variable by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/46
* Clarifying pedagogy by @beckynevin in https://github.com/lsst-epo/citizen-science-notebooks/pull/48
* Epo 8274 by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/49
* Epo 8318 by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/51
* Updated path to key by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/54
* adding details in README file by @beckynevin in https://github.com/lsst-epo/citizen-science-notebooks/pull/50
* Temporarily removing tabular notebook by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/55
* Updated the .gitignore based on pcw feedback by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/56
* Removed depr notebooks, pip install citsci package by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/52
* Referencing new package and import namespace by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/65
* Added code to clas. cell to render panda df by @ericdrosas87 in https://github.com/lsst-epo/citizen-science-notebooks/pull/72

## New Contributors
* @clareh made their first contribution in https://github.com/lsst-epo/citizen-science-notebooks/pull/2
* @beckynevin made their first contribution in https://github.com/lsst-epo/citizen-science-notebooks/pull/33

**Full Changelog**: https://github.com/lsst-epo/citizen-science-notebooks/commits/v0.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/lsst-epo/citizen-science-notebooks/releases/tag/v0.1.0'>v0.1.0</a>.</em>

**Comment by ericdrosas87** [comment]
@clareh @bnord @jsv1206 @beckynevin I am going to start using releases to denote when major functionality has been implemented. In the case of this release, Clare and I have been chatting with Chris of Zooniverse about how best to audit and detect similar data across Zooniverse projects. 

In this release, you may notice additional output in the `send_image_data()` cell:
<img width="876" alt="Screen Shot 2023-09-11 at 8 30 55 AM" src="https://github.com/lsst-epo/citizen-science-notebooks/assets/43484316/42ad89c8-9c53-4c8b-bfb3-6ebaa1222e5d">

If a match has been found - based on object ID - between the data you just sent to Zooniverse and data that has already been sent to Zooniverse for _other_ projects, you will receive a message indicating between 0 and 1 the percentage match.

This is still very much a work-in-progress and feedback is welcome and anticipated, and I plan to iterate on the development of this feature moving forward.

Let me know if you have any questions!

**Reply by jsv1206** [reply]
Any reason to have a percentage match instead of 0/1 ? Is this because it is looking for matches for all the object IDs within a dataset being sent?

Should we check for matches among the different subject sets within the same project ? This is in case we are planning to send data in chunks into the same project.

**Reply by ericdrosas87** [reply]
> Any reason to have a percentage match instead of 0/1 ? Is this because it is looking for matches for all the object IDs within a dataset being sent?

Correct - it's looking for any matches on object ID in the current dataset (subject set) being transferred against all data already sent to the Zooniverse using the citSci pipeline - _except_ for the current RSP user's previous datasets.

> Should we check for matches among the different subject sets within the same project ?

There's no reason why the code can't be changed to also match against the current user's previous datasets, I think it's a UX question - is that useful to the PI? etc.

> This is in case we are planning to send data in chunks into the same project.

I defer to what @clareh, @bnord, @beckynevin and yourself think is most useful and I can code as such.
